microsetta_public_api/api/diversity/alpha.py

```diff
@@ -32,26 +33,51 @@ def alpha_group(body, alpha_metric, summary_statistics=True,
             error=400, text='Either `summary_statistics`, `return_raw`, '
                             'or both are required to be true.'
             ), 400
+    if ('sample_ids' in body and 'metadata_query' in body) and \
```

Comment:
Can this be enforced by the openapi spec?

Reply:
Probably with `oneOf`... Let me look into it.

Reply:
Done in https://github.com/biocore/microsetta-public-api/pull/54/commits/3da6fcdc1047ee6ce3b9becb2270de5fcbf918e7